adoc/admin-ses-integration.adoc

```diff
@@ -87,36 +91,45 @@ spec:
       readOnly: false
 *EOF*
 ----
-. Verify that the pod exists and its status: 
+//TODO: Where do I get the input for POD_NAME, CONTAINER_NAME and IMAGE_NAME?
+//Can I just input any names I like?
+//Where do i get information for RBD_POOL and the monitors and their ports?
```

Comment:
RBD_POOL would be the pool name pre-created by user. Could reference to [create-a-block-device-pool](https://docs.ceph.com/docs/mimic/rbd/rados-rbd-cmds/#create-a-block-device-pool).

Reply:
MONITOR_IP are the monitor node address of SES cluster.
MONITOR_PORT would be the exposed monitor node port, default is 6789.
[L61](https://github.com/SUSE/doc-caasp/pull/400/files#diff-9144488edc4846ace018b6d59eabd176R61)